### PR TITLE
VPC flowlogs

### PR DESCRIPTION
**Pull request by paulschwarzenberger**

### CODE REVIEW

File: modules/keycloak/flowlogs.tf [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    actions = ["sts:AssumeRole"]
+    principals {
+      type        = "Service"
+      identifiers = ["vpc-flow-logs.amazonaws.com"]
+    }
+  }
+}
+
+resource "aws_iam_policy" "keycloak_flowlog_policy" {
+  name   = "keycloakkeycloakFlowlogPolicy${title(var.environment)}"
```
Comment (suzannehamilton) [inline modules/keycloak/flowlogs.tf]: Is "keycloakkeycloak" a typo?

File: modules/keycloak/flowlogs.tf [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+      "logs:DescribeLogGroups",
+      "logs:DescribeLogStreams"
+    ]
+
+    resources = [
+      "*",
```
Comment (suzannehamilton) [inline modules/keycloak/flowlogs.tf]: Does it make sense to tie this to the specific cloudwatch log group?

Ditto for the policy below.

File: modules/keycloak/flowlogs.tf (hunk 1 above)
Comment (suzannehamilton) [reply modules/keycloak/flowlogs.tf]: Sorry, out-of-date comment.

File: modules/keycloak/flowlogs.tf (hunk 2 above)
Comment (paulschwarzenberger) [reply modules/keycloak/flowlogs.tf]: Thanks, good idea, updated

**Pull request merged**